test/cloudant.test.js

```diff
@@ -113,7 +143,7 @@ describe('cloudant connector', function() {
       });
   });
 
-  describe('updateAll and updateAttributes', function() {
+  describeskip('updateAll and updateAttributes', function() {
```

Comment:
@raymondfeng I enable `updateAll and updateAttributes` and make it pass on local too.